aiohttp/server.py

```diff
@@ -388,3 +391,19 @@ def handle_request(self, message, payload):
         self.log_access(message, None, response, self._loop.time() - now)
 
         return drain
+
+    @property
+    def tcp_nodelay(self):
+        return self._tcp_nodelay
+
+    def set_tcp_nodelay(self, nodelay):
+        if nodelay == self._tcp_nodelay:
+            return
+        self._tcp_nodelay = nodelay
+        if self.transport is None:
+            # transport is closed
+            return
+        sock = self.transport.get_extra_info('socket')
+        if sock.family not in (socket.AF_INET, socket.AF_INET6):
```

Comment:
Transports need not have a `socket`, i.e. `get_extra_info('socket')` might return a `None`.
So, should probably do a ```if sock and sock.family not in (socket.AF_INET, socket.AF_INET6)``` to be on the safe side.